WordPress/WordPress.xcodeproj/project.pbxproj

```diff
@@ -363,7 +363,7 @@
 		24B1AE3124FEC79900B9F334 /* RemoteFeatureFlagTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 24B1AE3024FEC79900B9F334 /* RemoteFeatureFlagTests.swift */; };
 		24C69A8B2612421900312D9A /* UserSettingsTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 24C69A8A2612421900312D9A /* UserSettingsTests.swift */; };
 		24C69AC22612467C00312D9A /* UserSettingsTestsObjc.m in Sources */ = {isa = PBXBuildFile; fileRef = 24C69AC12612467C00312D9A /* UserSettingsTestsObjc.m */; };
-		24CE2EB1258D687A0000C297 /* WordPressFlux in Frameworks */ = {isa = PBXBuildFile; productRef = 24CE2EB0258D687A0000C297 /* WordPressFlux */; };
```

Comment:
What are these changes for? 
Asking because they look to revert the changes from https://github.com/wordpress-mobile/WordPress-iOS/pull/16351/files, which was merged a couple of months ago. There's some context around that PR (internal reference p1619035833107300-slack-C02Q9GACF), and IIRC it has to do with changes different versions of Xcode apply to the project file.

Reply:
These changes were the result of running `rake dependencies` after updating the gutenberg reference. I tried resetting `project.pbxproj` back to the `develop` version and running `rake dependencies` and the same changes appeared again. I'm using Xcode 12.4.

Next I tried moving a file up and back down in the Project navigator in Xcode and a similar diff happened as in https://github.com/wordpress-mobile/WordPress-iOS/pull/16351. I'm not sure why this is happening to us, but running `rake dependencies` now doesn't create the same diff anymore. I guess I'll go ahead and commit these changes.

Reply:
Ok, they look like harmless changes, I was only concerned that they would undo https://github.com/wordpress-mobile/WordPress-iOS/pull/16351/files, which was also made with Xcode 12.4.
Since the changes have now been removed from this PR, I'll mark this done.